src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotificationsTest.java

```diff
@@ -0,0 +1,202 @@
+package com.cloudbees.jenkins.plugins.bitbucket;
```

Comment:
Did you try to use a JenkinsRule instead to mock whoke stack? In this manner the test is easier to mantain (you don't need powermockito).
There is also a factory to register a mock api client. Have a look intorno test package

Reply:
I tried, but I don't remember where I couldn't use it.
I'll try to take a look at this one more time this week.